planet/cli/data.py

```diff
@@ -64,11 +64,12 @@ def assets_to_filter(ctx, param, assets: List[str]) -> Optional[dict]:
 
 def check_item_types(ctx, param, item_types) -> Optional[List[dict]]:
     # Set difference between given item types and all item types
-    set_diff = set([item.lower() for item in item_types]) - set(
+    invalid_item_types = set([item.lower() for item in item_types]) - set(
```

Comment:
Can we somehow change the capitalization of the user-specified item-type to match the capitalization in ALL_ITEM_TYPES? I think there is code for that in the specs module. If we don't change it, the item capitalization will stay how the user specified it, but the API will not accept capitalization that doesn't exactly match what is in the spec.

Reply:
Oh yeah, in `specs.validate_item_type()`!